src/main/java/com/fasterxml/jackson/databind/util/LRUMap.java

```diff
@@ -25,29 +24,26 @@
     implements LookupCache<K,V>, // since 2.12
         java.io.Serializable
 {
-    private static final long serialVersionUID = 1L;
+    private static final long serialVersionUID = 2L;
 
-    protected final transient int _maxEntries;
+    protected final int _initialEntries;
+    protected final int _maxEntries;
+    protected final transient ConcurrentLinkedHashMap<K,V> _map;
 
-    protected final transient ConcurrentHashMap<K,V> _map;
-    
     public LRUMap(int initialEntries, int maxEntries)
     {
-        // We'll use concurrency level of 4, seems reasonable
-        _map = new ConcurrentHashMap<K,V>(initialEntries, 0.8f, 4);
+        _initialEntries = initialEntries;
         _maxEntries = maxEntries;
+        // We'll use concurrency level of 4, seems reasonable
+        _map = new ConcurrentLinkedHashMap.Builder<K, V>()
+                .initialCapacity(initialEntries)
+                .maximumWeightedCapacity(maxEntries)
+                .concurrencyLevel(4)
```

Comment:
So, was this something that did not matter any more, @ben-manes ?

Reply:
Yes, this has no effect and can be removed if misleading

Reply:
I've removed this setting now